### DISCUSSION TITLE
How long does it take to run SeisSol Examples

### DISCUSSION THREAD

**Discussion by smallriver666** [SeisSol/SeisSol#560] [Q&A] [answered]
I'm running the LOH1 example, but as shown in the figure below, it doesn't seem to respond here
![image](https://user-images.githubusercontent.com/104721358/171162597-631237b4-e298-4dd9-97a8-9fd586602cf0.png)
in output file, I get these files,
![image](https://user-images.githubusercontent.com/104721358/171163075-0988e97a-8afc-4982-8f9e-c21c11111253.png)
and I have changed the loh.geo, show inthe figure below
![image](https://user-images.githubusercontent.com/104721358/171163554-182d85ec-4720-47d9-8755-bab10702e58c.png)
![image](https://user-images.githubusercontent.com/104721358/171163609-c8eb5dfe-de5a-4064-bbc7-defde7c4d71f.png)
` Start calc_SeisSol ...` need much time?? I run the Example loh1 the first time.
and my centos7's configurations are `48G running memory, 8-core cpu`

**Comment by Thomas-Ulrich** [comment, marked as answer]
you can try to change 
```
ReceiverOutputInterval = 10.0
```
in the parameters.par to something very small (e.g. 1e-3) and then you will get more often receivers buffers get emptied.

**Reply by smallriver666** [reply]
> you can try to change
> 
> ```
> ReceiverOutputInterval = 10.0
> ```
> 
> in the parameters.par to something very small (e.g. 1e-3) and then you will get more often receivers buffers get emptied.

Thank you very much, Following your prompts, I can see the results of the run respond, but should this process take a lot of time?
